src/components/attendees/AttendeeRow.tsx

```diff
@@ -1,23 +1,30 @@
-import './styles.css';
 import { Attendee } from '../../models/attendee';
 import AttendeeCheckBox from './AttendeeCheckBox';
 import { ReactComponent as EmailIcon } from '../../assets/Attendees/google.svg';
 import { ReactComponent as GoogleIcon } from '../../assets/Attendees/email.svg';
+import { ReactComponent as OpenIcon } from '../../assets/Attendees/open.svg';
+import './styles.css';
+
 /**
  * @returns row for an attendee in the attendee table.
  */
 
-interface AttendeeObject {
+export default function AttendeeRow(props: {
```

Comment:
There are a lot of things being pulled out of props below. As a suggestion:
```
{ popup, attendee }: {
    popup: {
        /*blah blah*/
    },
    attendee: Attendee,
    setOpen: (isOpen: boolean) => void
}
```

That way you might not have to unbundle it so many times.